src/Messages/Interactions/TSMessage.m

```diff
@@ -132,6 +133,10 @@ - (nullable instancetype)initWithCoder:(NSCoder *)coder
     }
 
     _schemaVersion = OWSMessageSchemaVersion;
+
+    // We _DO NOT_ set _receivedAt_ in this constructor.  We don't want to
+    // set the receivedAt time for old messages in the data store.
```

Comment:
`receivedAt` is superceded by `receivedAtDate`.  I suppose we could continue to set it, but it would serve no value - and possibly create confusion.  The message would then have two timestamps that were always _almost_ the same but always off by a few ms.  We would never want to consult `receivedAt`, so why set it?